modules/nw-dual-stack-convert.adoc

```diff
@@ -74,7 +74,9 @@ $ oc patch network.config.openshift.io cluster \// <1>
 network.config.openshift.io/cluster patched
 ----
 
-. Specify IPv6 VIPs for API and Ingress services for your cluster. Create a YAML configuration patch file that has a similar configuration to the following example:
+. On installer-provisioned infrastructure where you added IPv6 VIPs for API and Ingress services, complete the following steps:
```

Comment:
```suggestion
. On the installer-provisioned infrastructure where you added IPv6 VIPs for API and Ingress services, complete the following steps:
```

Reply:
This step could be confusing if you don't read the intro paragraphs (I know, `cause I didn't read the intro paragraphs.). Would it make sense to make the _On installer-provisioned infrastructure only_ into a prerequisite? A bit of a long prerequisite, I know.

Reply:
Unfortunetly not, as some steps are also for UPI and we don't use conditionals in the file as the networking docs are consolidated.